packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -707,6 +707,44 @@ export const guides = [
           },
         ],
       },
+      {
+        title: "Automate managing with Terraform",
+        guideName: "terraform-landing-page",
+        subLinks: [
+          {
+            title: "Terraform overview",
+            guideName: "terraform-overview",
+          },
+          {
+            title: "Enable Terraform access",
+            guideName: "terraform-enable-org-access",
+          },
+          {
+            title: "Manage user access",
+            guideName: "terraform-manage-user-access",
+          },
+          {
+            title: "Manage groups",
+            guideName: "terraform-manage-groups",
+          },
+          {
+            title: "Manage authentication services",
+            guideName: "terraform-manage-external-authenticators",
+          },
+          {
+            title: "Manager end-user experience",
+            guideName: "terraform-manage-end-user-experience",
+          },
+          {
+            title: "Design secure Terraform access",
```

Comment:
How come the title here and the h1 in the guide are different?

Reply:
Good catch! The next commit fixes this (I hope).